include/picongpu/particles/ionization/byField/Keldysh/AlgorithmKeldysh.hpp

```diff
@@ -57,10 +58,10 @@ namespace ionization
          * \param parentIon particle instance to be ionized with position at t=0 and momentum at t=-1/2
          * \param randNr random number, equally distributed in range [0.:1.0]
          *
-         * \return number of new macro electrons to be created
+         * \return number of new macro electrons to be created and ionization energy
```

Comment:
optional: it would be better to state first the energy and then the number of electrons as you return the values.